app/views/admin/facility_groups/_form.html.erb

```diff
@@ -37,8 +37,10 @@
         <%= form.select(:protocol_id, protocols.order(:name).map { |protocol| [protocol.name, protocol.id] }) %>
       <% end %>
 
-      <%= form.number_field :district_estimated_population, label: "Estimated hypertensive population", autocomplete: "off", 
-        help: "Leave blank if unknown" %>
+      <% if Flipper.enabled?(:estimated_population) %>
```

Comment:
you probably want `current_admin.feature_enabled?(:estimated_population)` here so the feature flag check is at the user level.